.github/workflows/distribute-grain.yml

```diff
@@ -0,0 +1,65 @@
+name: Distribute Grain
+on:
+  # A new Cred interval every Sunday. We'll distribute Grain 5 minutes
+  # after the new interval
+  schedule:
+    - cron: 5 0 * * 0 # 00:05 UTC, 16:05 PST
+  push:
+    branches:
+      - "graintest-*" # won't actually update ledger or deploy frontend
+
+jobs:
+  distribute-grain:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "[debug] Check GitHub event name"
+        # This is used to determine whether we actually deploy to GitHub
+        # Pages, so we print it upfront to aid with debugging.
+        run: echo ${{ github.event_name }}
+
+      - name: Install Packages 🔧
+        run: |
+          cd calc
```

Comment:
@topocount what's with all the `cd calc`s?

Reply:
removed